backend/datahubhel/ksql_scripts/load_ta120_connectors.py

```diff
@@ -0,0 +1,86 @@
+import logging
+from typing import List
+from urllib.parse import urlparse
+
+import requests
+from django.conf import settings
+
+from .http_response_check import check_errors
+
+LOG = logging.getLogger(__name__)
+
+SINK_CONFIGURATIONS = {
+    "observation-sink": {
+        "connector.class": "io.confluent.connect.jdbc.JdbcSinkConnector",
+        "topics.regex": r"fi\.fvh\.observations\.\w+\.\w+",
+        "table.name.format": settings.SINK_TABLE_NAME,
+        "key.converter": "org.apache.kafka.connect.storage.StringConverter",
+        "value.converter": "io.confluent.connect.avro.AvroConverter",
+        "value.converter.schema.registry.url": settings.SCHEMA_REGISTRY_URL,
+        "connection.url": "jdbc:" + settings.SINK_DATABASE_URL,
+        "connection.user": settings.SINK_DB_USERNAME,
+        "connection.password": settings.SINK_DB_PASSWORD,
```

Comment:
These three should come from the `settings.DATABASE_URL`. Is it really needed to have user and password separately or can they be given in the single URL as with Django?

Reply:
I tried it and connector task stopped immediately. So, it seems like we actually need to have username and password separately.